spring-cloud-generator/generate-all.sh

```diff
@@ -12,14 +18,23 @@ cd spring-cloud-generator
 
 bash download-repos.sh
 libraries=$(cat $WORKING_DIR/library_list.txt | tail -n+2)
-while IFS=, read -r library_name googleapis_location coordinates_version; do
+while IFS=, read -r library_name googleapis_location coordinates_version googleapis_commitish; do
   echo "processing library $library_name"
   group_id=$(echo $coordinates_version | cut -f1 -d:)
   artifact_id=$(echo $coordinates_version | cut -f2 -d:)
-  version=$(echo $coordinates_version | cut -f3 -d:)
-  bash $WORKING_DIR/generate-one.sh -c $library_name -v $version -i $artifact_id -g $group_id -p $PROJECT_VERSION -f $googleapis_location
+  bash $WORKING_DIR/generate-one.sh -c $library_name -v $PROJECT_VERSION -i $artifact_id -g $group_id -p $PROJECT_VERSION -f $googleapis_location -x $googleapis_commitish
```

Comment:
I think `-v $PROJECT_VERSION` is redundant info and is not be needed anymore?

Reply:
Yes, it is redundant and is removed by https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1430. It's still necessary as an argument to `-p` (just confirmed)